create.go

```diff
@@ -166,6 +166,8 @@ func (r *Resource) computeCreateEventPatches(ctx context.Context, obj interface{
 				Path:  "/status/cluster/conditions",
 				Value: clusterStatus.WithCreatingCondition(),
 			})
+
+			r.logger.LogCtx(ctx, "level", "info", "message", fmt.Sprintf("setting %#q status condition", providerv1alpha1.StatusClusterTypeCreating))
```

Comment:
Cool those logs are `info` level. I think we should emit `info` when we do actual changes.

Reply:
This was a request from @marians in the PM so maybe please sort it with him. I do not mind either way.